astroquery/jplhorizons/core.py

```diff
@@ -110,19 +108,9 @@ def __init__(self, id=None, location=None, epochs=None,
         self.epochs = epochs
 
         # check for id_type
-        if id_type is None:
-            warnings.warn('id_type was not specified, defaulting to a '
-            '"smallbody" search.  This behavior is deprecated and will be '
-            'changed to "" (empty string / Horizons default) in the future.',
-            AstropyDeprecationWarning)
-            id_type = 'smallbody'
-        elif id_type == 'majorbody':
```

Comment:
sorry, I was confusing. I think it's great to raise the warning for `majorbody`, I only had problem with giving out a warning with the default behaviour.

Reply:
Oops!  I'll restore it.